README.md

```diff
@@ -1 +1,87 @@
+```sh
+docker compose -f docker-compose.yml -f docker-compose-htsget.yml up -d
+```
+
+The logs for the two docker compose files can be accessed using the following commands for storage-and-interfaces and htsget respectively
+```sh
+docker logs -f docker-compose.yml -f
+docker logs -f docker-compose-htsget.yml -f
+```
+
+## Access data with htsget
+In order to test the htsget implementation, there needs to be some data ingested in the archive. In case you do not have any data to ingest, follow the procedure in the [add data section](#(optional)-add-data) before continuing in this section.
+
+## Get file information
+**NOTE:** The sections below assume that data has been ingested in the archive.
+
+In order to make sure there exist files in the database and in order to get the ids needed in order to request one of them using htsget execute in the database container using
+```sh
+docker exec -it postgres bash
+```
+Connect to the postgres database using
+```sh
+psql -h localhost -U postgres lega
+```
+with password `rootpass` and run the following query
+```sh
+select datasets.stable_id, files.stable_id, submission_file_path from sda.file_dataset join sda.files on file_id = files.id full join sda.datasets on file_dataset.id = sda.datasets.id;
+```
+You should be able to see some files and the dataset they have been added to. This information will be used in the next step in order to request one of the files.
+
+## Request a file with samtools
+
+Now that all the information regarding the ingested files is available, it should be possible to access a file using the htsget protocol. One way to do that, is using samtools, included in the `docker-compose-htsget.yml` file.
+
+Start by executing in the samtools container with
+```sh
+docker exec -it samtools-client bash
+```
+1. The samtools client requires two variable to be exported, while a token containing the visas for the specified user should be provided. Set the two environment variables required using
+```sh
+export HTS_ALLOW_UNENCRYPTED_AUTHORIZATION_HEADER="I understand the risks"
+export HTS_AUTH_LOCATION=token.txt
+```
+2. Get the token from the OIDC endpoint by running
+```sh
+echo $(curl -k -S https://oidc:8080/tokens | jq '.[0]') > token.txt
+```
+3. Finally get the file using
+```sh
+samtools view http://server:3000/reads/s3/<dataset_id>/<file_path>
+```
+where `dataset_id` and `file_path` are the results from the query to the database.
+
+### (Optional) Add data
+In case you do not have any data to ingest, there exists a [script](https://github.com/GenomicDataInfrastructure/starter-kit-storage-and-interfaces/blob/main/scripts/load_data.sh) that can load a very small sample dataset in the storage-and-interface repository, which can be executed using the same docker compose file. 
+
+To run the container executing this script run
+```sh
+docker compose up data_loader
+```
+Once the script is finished and the data should be loaded. 
+
```

Comment:
This need to be more general since some of it needs to be in the `starter-kit` repo